jasc/.bazelrc

```diff
@@ -0,0 +1,18 @@
+build --announce_rc
+
+build --experimental_repo_remote_exec
+build --cxxopt=-std=c++17 --host_cxxopt=-std=c++17
+build --cxxopt=-w --host_cxxopt=-w
+build --define=grpc_no_ares=true
+build --define=tsl_link_protobuf=true
+build --define open_source_build=true
+
+build --define framework_shared_object=true
+build --define tsl_protobuf_header_only=true
+build --define=use_fast_cpp_protos=true
+build --define=allow_oversize_protos=true
+
+build --copt=-DMLIR_PYTHON_PACKAGE_PREFIX=jaxlib.mlir.
+
+build -c opt
```

Comment:
Do we really want to force opt builds by default?

Reply:
Enyzme-JAX is not the best source of truth, I'm afraid. It also copied these from random places. Let's at least document it was borrowed from there so we can later try and figure out why these are necessary. And let's try removing `-c opt` as it increases compilation time. I suspect this may be to force XLA to enable some optimizations in its compiler, but we are better off without it IMO.